docs/src/charm/reference/versions/1.32.md

```diff
@@ -0,0 +1,106 @@
+charm config [charm config].
+- **Terraform Modules** - add basic Terraform modules for the {{product}} 
+charms [#194].
+- **Upgrade Orchestration** - introduce upgrade orchestration for control 
+plane nodes [#200].
+- **Multiple Worker Integration** - allow the `k8s` charm to integrate with 
+multiple `k8s-worker` units [#221].
+
+## Fixes
+
+- Worker goes into error after control plane departed [#75][issue #75]
+- Control plane does not go into blocked when no relation to worker 
+[#90][issue #90]
+- Enable and configure `Loadbalancer` feature [#109][issue #109]
+- Enable and configure `LocalStorage` feature [#110][issue #110]
+- Support to configure Custom registries [#111][issue #111]
+- Ability to configure Cilium option `--vlan-bpf-bypass` [#112][issue #112]
+
+
+## Deprecations and API changes
+
+- Upstream
+
+For details of other deprecation notices and API changes for Kubernetes 1.32, 
+please see the
+relevant sections of the [upstream release notes][upstream-changelog-1.32].
+
+[upstream-changelog-1.32]: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.32.md#deprecation
+
+## Also in this release
+
+- Replace `AssertionError` with `ReconcileError` [#173]
+- Renames `annotations` to `cluster-annotations` in charm config [#198]
+- Add `kube-apiserver-extra-sans` option [#201]
+- Add worker `bootstrap-node-taints` setting [#215]
+- Enhance status visibility during cluster upgrades [#216]
+
+## Contributors
+
+Many thanks to [@addyess], [@mateoflorido], [@bschimke95], [@louiseschmidtgen],
+[@eaudetcobello], [@berkayoz], [@HomayoonAlimohammadi], [@ktsakalozos],
+[@kwmonroe], [@maci3jka], [@petrutlucian94], [@evilnick], [@nhennigan],
+[@perk], [@asbalderson].
+
+<!-- LINKS -->
+<!--     PR     -->
+[#118]: https://github.com/canonical/k8s-operator/pull/118
+[#149]: https://github.com/canonical/k8s-operator/pull/149
+[#166]: https://github.com/canonical/k8s-operator/pull/166
+[#173]: https://github.com/canonical/k8s-operator/pull/173
+[#194]: https://github.com/canonical/k8s-operator/pull/194
+[#198]: https://github.com/canonical/k8s-operator/pull/198
+[#200]: https://github.com/canonical/k8s-operator/pull/200
+[#201]: https://github.com/canonical/k8s-operator/pull/201
+[#215]: https://github.com/canonical/k8s-operator/pull/215
+[#216]: https://github.com/canonical/k8s-operator/pull/216
+[#221]: https://github.com/canonical/k8s-operator/pull/221
+<!--     ISSUE      -->
+[issue #75]: https://github.com/canonical/k8s-operator/issues/75
+[issue #90]: https://github.com/canonical/k8s-operator/issues/90
+[issue #109]: https://github.com/canonical/k8s-operator/issues/109
+[issue #110]: https://github.com/canonical/k8s-operator/issues/110
+[issue #111]: https://github.com/canonical/k8s-operator/issues/111
+[issue #112]: https://github.com/canonical/k8s-operator/issues/112
+<!--     MISC       -->
+[charm config]: /src/charm/reference/charm-config.md
```

Comment:
I'm not sure which file you are trying to link here

Reply:
I was initially planning to add a charm-config.md file to showcase all the options that we can provide to the charm. But I think it's automatically available at CharmHub. Let me update this part.

Reply:
Done

Reply:
I think it is a good idea but we don't want to link to an empty page. How about we link to charmhub for now and when we have this config page we can link there?